### PR TITLE
Add eslint setup instructions

### LINKED ISSUE #4: Manual on how to use in another module
Really happy we have this now! 😍 

I wanted to add it to https://github.com/orbitdb/orbit-db-identity-provider to get in to use quickly, but I realized there's no instructions how to use it in a module. Nothing ever being obvious to a user, it'd be best to document each step and command in detail on how to add this (and linting and other tools, if assumed to be installed) to a project.

### PR DESCRIPTION
**Pull request by thiagodelgado111**
I've added some info on how to setup `eslint` for other projects from scratch here pointing to their `getting-started`. Is that better @haadcode? :)

Closes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# eslint-config-orbitdb
+
+The eslint configuration used for orbit-db projects
+
+## Usage
```
Comment (RichardLitt) [inline README.md]: This isn't Usage,  but an Install section, it looks like?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+If you use yarn, run `npm info "@orbitdb/eslint-config-orbitdb@latest" peerDependencies` to list the peer dependencies and versions, then run `yarn add --dev <dependency>@<version>` for each listed peer dependency. See below for npm instructions.
+
+1. Install the correct versions of each package, which are listed by the command:
+
+  ```sh
+  npm info "@orbitdb/eslint-config-orbitdb@latest" peerDependencies
+  ```
+
+  Linux/OSX users can run
```
Comment (RichardLitt) [inline README.md]: Add colon

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Pretty good. I have a few comments, and some fixes to push later. (See https://github.com/orbitdb/repo-template for what those are, if you want to do them, here?)

**Comment by thiagodelgado111** [comment]
Fixed and added the missing files, should be good to go now :) cc @RichardLitt

**Comment by haadcode** [comment]
I tried setting it up again and while got closer, noticed a few ways we can improve the developer experience. 

I personally find it really nice to try out a repo when the installation and usage instructions are clear and explicit, so I wrote the step-by-step instructions to the README in #7 and PRed a change to the deps in #6 (required for #7 to work as written). 

In #7 I also removed the unrelated files (COC, contributing guidelines, npmignore) to clean up this PR ("Add eslint setup instructions"). We should create separate PRs for adding those.

If you like the modified version of the README in #7, feel free to merge it here and we can merge this to master.

**Comment by RichardLitt** [comment]
@haadcode When you get a chance, take a look at @thiagodelgado111's comments in #6 and #7.

**Comment by RichardLitt** [comment]
Going with standard for now. Closing this for the moment.

**Pull request closed without merging**